content/en/storage-providers/curio/setup.md

```diff
@@ -0,0 +1,104 @@
+    tar xvfz yugabyte-2.20.0.1-b1-linux-x86_64.tar.gz && cd yugabyte-2.20.0.1/
+    ./bin/post_install.sh
+    ./bin/yugabyted start --advertise_address 127.0.0.1  --master_flags rpc_bind_addresses=127.0.0.1 --tserver_flags rpc_bind_addresses=127.0.0.1
+    ```
+    ```
+    +----------------------------------------------------------------------------------------------------------+
+    |                                                yugabyted                                                 |
+    +----------------------------------------------------------------------------------------------------------+
+    | Status              :                                                                                    |
+    | Replication Factor  : None                                                                               |
+    | YugabyteDB UI       : http://127.0.0.1:15433                                                             |
+    | JDBC                : jdbc:postgresql://127.0.0.1:5433/yugabyte?user=yugabyte&password=yugabyte          |
+    | YSQL                : bin/ysqlsh   -U yugabyte -d yugabyte                                               |
+    | YCQL                : bin/ycqlsh   -u cassandra                                                          |
+    | Data Dir            : /root/var/data                                                                     |
+    | Log Dir             : /root/var/logs                                                                     |
+    | Universe UUID       : 411422ee-4c17-4f33-996e-ced847d10f5c                                               |
+    +----------------------------------------------------------------------------------------------------------+
+    ```
+
+You can adjust the `--advertise_address`, `--rpc_bind_addresses` and `--tserver_flags` according to your own configuration and needs.
+
+## Migrating from Lotus-miner to Curio
+Curio provides a utility to users onboard quickly. Please run the below command on your `lotus-miner` node and follow the os-screen instructions.
+
+```shell
+curio guided-setup
+```
+
+Once the migration is complete, you can shut down all of your workers and miner processes. You can start `curio` process to replace them with correct [configuration layer]({{< relref "config" >}}).
+
+### Testing the setup
+
+You can confirm that the `curio` process is able to schedule and compute WindowPoSt by running a WindowPoSt test computation:
+
+```shell with-output
+curio test window-post task
+```
+
+From the output we can confirm that a WindowPoSt gets inserted to the database, and is being picked up by the Curio process running with the *wdpost* configuration layer.
+
+## Initiating a new Curio cluster
+Curio provides a utility to users onboard quickly. Please run the below command on your new Curio node and follow the os-screen instructions.
+
+```shell
+curio
```

Comment:
sptool new miner ?
curio guided-setup? 
I'm not sure what you are going for here. It needs more documentation.

Reply:
My bad. I have fixed the command.